docs/ops/state_backends.md

```diff
@@ -123,8 +123,7 @@ RocksDBStateBackend is currently the only backend that offers incremental checkp
 
 ## Configuring a State Backend
 
-State backends can be configured per job. In addition, you can define a default state backend to be used when the
-job does not explicitly define a state backend.
+State backends can be configured per job in code. In addition, you can define a default state backend in **flink-conf.yaml** that is used when the job does not explicitly define a state backend.
```

Comment:
@zentol I find that "in code" reads rather awkwardly, and I don't see how it adds any value, since the details of how to do per-job configuration are shown below. Nevertheless, this topic can be a bit confusing, so I would suggest something more like this (assuming I got the details right):

The default state backend, if you specify nothing, is the jobmanager. If you wish to establish a different default for all jobs on your cluster, you can do so by defining a new default state backend in **flink-conf.yaml**. The default state backend can be overridden on a per-job basis, as shown below.